package/src/components/ImageGallery/components/ImageGalleryFooter.tsx

```diff
@@ -27,13 +29,16 @@ import {
   VideoType,
 } from '../../../native';
 
+import { FileTypes } from '../../../types/types';
+import { getReactNativeVersion } from '../../../utils/getReactNativeVersion';
+import type { Photo } from '../ImageGallery';
+
+const SafeAreaView = getReactNativeVersion().minor >= 81 ? SafeAreaViewOriginal : RNSafeAreaView;
```

Comment:
We shouldn't check the version. Ideally, we should have this:
```suggestion
const SafeAreaView = RNSafeAreaView ?? SafeAreaViewOriginal;
```

Reply:
Yes, it covers all cases, but what I don't think we should check against the actual RN version. Instead, let's make our code always use the external one, and fall back to RN.